pkg/datastore/mysql/query.go

```diff
@@ -74,6 +74,8 @@ func buildWhereClause(filters []datastore.ListFilter) string {
 			// Make string of (?,...) which contains the number of `?` equal to the element number of filter.Value
 			valLength := reflect.ValueOf(filter.Value).Len()
 			conds[i] = fmt.Sprintf("%s %s (?%s)", filter.Field, filter.Operator, strings.Repeat(",?", valLength-1))
+		case "JSON_CONTAINS":
+			conds[i] = fmt.Sprintf("%s(%s, '%q', '$')", filter.Operator, filter.Field, filter.Value)
```

Comment:
We shouldn't pass the `filter.Value` here, it will automatically be passed by [here](https://github.com/pipe-cd/pipe/blob/master/pkg/datastore/mysql/mysql.go#L98), what we do here is to make the string query that mark every value on the query as `?` character. 😄